lib/src/rbac/default_role_manager.dart

```diff
@@ -4,15 +4,16 @@ import 'package:casbin/src/rbac/role.dart';
 import 'package:casbin/src/rbac/role_manager.dart';
 
 class DefaultRoleManager implements RoleManager {
-  HashMap<String, Role> allRoles;
+  Map<String, Role> allRoles;
   int maxHierarchyLevel;
 
   /// DefaultRoleManager is the constructor for creating an instance of the
   /// default RoleManager implementation.
   ///
   /// [maxHierarchyLevel] the maximized allowed RBAC hierarchy level.
   DefaultRoleManager(int maxHierarchyLevel) {
-    this.maxHierarchyLevel = maxHierarchyLevel;
+    this.maxHierarchyLevel = maxHierarchyLevel ?? 10;
```

Comment:
Why 10?

Reply:
@KNawm It's set to 10 in jCasbin and node-casbin

Reply:
@KNawm It's the default value if maxHierarchyLevel is not provided.

Reply:
Yes, we set default maxHierarchyLevel to 10. This one would be OK.